Add news item for Einstein Probe, Astro Flavored Markdown, and v4 of unified schema (#2289)
nasa-gcn/gcn.nasa.gov: app/routes/news._index.mdx

Reviewer: Rather than a long and deeply nested list, please break this up into subsections, like this...

```md

We are pleased to announce three new features today:

- [Einstein Probe Notices](#einstein-probe-notices)
- [Astro Flavored Markdown](#astro-flavored-markdown)
- [Notices Schema v4.0.0](#notices-schema-v4.0.0)

#### Einstein Probe Notices

...text here...

#### Astro Flavored Markdown

...text here...

#### Notice Schema v4.0.0

...text here...
```

## 2024

    <CollectionItem
      className="maxw-none"
      variantComponent={
        <CollectionCalendarDate datetime={'May 20, 2024'} />
      }
    >
      <CollectionHeading headingLevel="h3">
        <Anchor>Einstein Probe Notices, Astro Flavored Markdown for Circulars, and Notices Schema v4.0.0</Anchor>
      </CollectionHeading>
      <CollectionDescription>

>         - **Einstein Probe** new GCN notice type is now available in JSON format over [Kafka](/quickstart), but are not yet available via email.  They are not available via GCN Classic.

Reviewer: Don't put double spaces between sentences. They aren't rendered in HTML anyway.

```suggestion
        - **Einstein Probe** new GCN notice type is now available in JSON format over [Kafka](/quickstart), but are not yet available via email. They are not available via GCN Classic.
```

Reviewer: Just for consistency. It's best to pick one style or the other.

>         - **Einstein Probe** new GCN notice type is now available in JSON format over [Kafka](/quickstart), but are not yet available via email.  They are not available via GCN Classic.

Reviewer 2: @lpsinger when will the email channel be ready?

          - Einstein Probe (EP) Summary: EP is a mission of the [Chinese Academy of Sciences (CAS)](https://ep.bao.ac.cn/ep/), in collaboration with the [European Space Agency (ESA)](https://www.cosmos.esa.int/web/einstein-probe/mission) and the [Max Planck Institute for Extraterrestrial Physics (MPE)](https://www.mpe.mpg.de/7867826/EinsteinProbe), dedicated to time-domain high-energy astrophysics. Its primary goals are to discover high-energy transients and monitor variable objects.

>             - Occurance Rate: The EP notices occur at a rate of ~100 times per year.

Reviewer 2: @jracusin a typo here, should be `Occurrence Rate`

            - Time Delay: The post-event processing and communication of alert is distributed to GCN within ~60 seconds of detection.
            - Notice Type: The [new WXT notice type (`gcn.notices.einstein_probe.wxt.alert`)](/missions/einstein_probe) provides trigger information including localization.
            - Notice Schema and Examples: The JSON schema defining the notice format and examples can be viewed in the [Schema Browser](/docs/schema) or the GCN [schema repository](https://github.com/nasa-gcn/gcn-schema).

>         - **Astro-Flavored Markdown** of GCN Circulars provides an enhanced experience in using our GCN Circulars submission web form.  It now optionally utilizes a custom form of Markdown, a lightweight markup language.  It allows embedding of data via tables, enriched text formatting, enhancing the already implemented automatic cross-referencing to other Circulars and common astronomy archives. [Documentation on Astro Flavored Markdown](/docs/circulars/markdown) is available.

Reviewer: - It's spelled "Astro Flavored Markdown", no hyphen, just like "GitHub Flavored Markdown".
- Remove double spaces between sentences.

```suggestion
        - **Astro Flavored Markdown** of GCN Circulars provides an enhanced experience in using our GCN Circulars submission web form. It now optionally utilizes a custom form of Markdown, a lightweight markup language.  It allows embedding of data via tables, enriched text formatting, enhancing the already implemented automatic cross-referencing to other Circulars and common astronomy archives. [Documentation on Astro Flavored Markdown](/docs/circulars/markdown) is available.
```

>         - **Notices Schema v4.0.0** for JSON-format GCN Notices is released.  Explore GCN Schema via the [Schema Browser](/docs/schema/v4.0.0/gcn/notices).  Changes to the GCN Schema include:

Reviewer: ```suggestion
        - **Notices Schema v4.0.0** for JSON-format GCN Notices is released. Explore GCN Schema via the [Schema Browser](/docs/schema/v4.0.0/gcn/notices). Changes to the GCN Schema include:
```

## 2024

    <CollectionItem
      className="maxw-none"
      variantComponent={
        <CollectionCalendarDate datetime={'May 20, 2024'} />
      }
    >
      <CollectionHeading headingLevel="h3">

>         <Anchor>New Einstein Probe Notices, Circulars Astro-Flavored Markdown, and Notices Schema v4.0.0</Anchor>

Reviewer: ```suggestion
        <Anchor>Einstein Probe Notices, Astro Flavored Markdown for Circulars, and Notices Schema v4.0.0</Anchor>
```

      </CollectionHeading>
      <CollectionDescription>

>         - **Einstein Probe** new GCN notice type is now available in JSON format over Kafka (https://gcn.nasa.gov/quickstart), but are not yet available via email.  They are not available via GCN Classic.

Reviewer: Please don't use bare URLs as hyperlinks. Add some link text.

>           - Einstein Probe (EP) Summary: EP is a mission of the Chinese Academy of Sciences (CAS), in collaboration with the European Space Agency (ESA) and the Max Planck Institute for extraterrestrial Physics (MPE), Germany, dedicated to time-domain high-energy astrophysics. Its primary goals are to discover high-energy transients and monitor variable objects.

Reviewer 3: ```suggestion
          - Einstein Probe (EP) Summary: EP is a mission of the Chinese Academy of Sciences (CAS), in collaboration with the European Space Agency (ESA) and the Max Planck Institute for Extraterrestrial Physics (MPE), Germany, dedicated to time-domain high-energy astrophysics. Its primary goals are to discover high-energy transients and monitor variable objects.
```

Reviewer: Add hyperlinks for all of these other agencies.

>             - Occurance Rate: The EP Mission notices occur at a rate of ~100 times per year.

Reviewer: ```suggestion
            - Occurance Rate: The EP notices occur at a rate of ~100 times per year.
```

            - Time Delay: The post-event processing and communication of alert is distributed to GCN within ~60 seconds of detection.

>             - Notice Type: The new WXT notice type (gcn.notices.einstein_probe.wxt.alert) provides trigger information including localization.

Reviewer: ```suggestion
            - Notice Type: The new WXT notice type (`gcn.notices.einstein_probe.wxt.alert`) provides trigger information including localization.
```

>             A more extensive description of the information provided in this new notice type is available at https://gcn.nasa.gov/missions/einstein-probe.

Reviewer: Add link text.

>             - Notice Schema and Examples: The JSON schema defining the notice format and examples can be viewed in the [Schema Browser](https://gcn.nasa.gov/docs/schema) or the GCN [schema repository](https://github.com/nasa-gcn/gcn-schema).

Reviewer: ```suggestion
            - Notice Schema and Examples: The JSON schema defining the notice format and examples can be viewed in the [Schema Browser](/docs/schema) or the GCN [schema repository](https://github.com/nasa-gcn/gcn-schema).
```

>         - **Astro-Flavored Markdown** of GCN Circulars provides an enhanced experience in using our GCN Circulars submission web form.  It now optionally utilizes a custom form of Markdown, a lightweight markup language.  It allows embedding of data via tables, enriched text formatting, enhancing the already implemented automatic cross-referencing to other Circulars and common astronomy archives. [Documentation on Astro Flavored Markdown](https://gcn.nasa.gov/docs/circulars/markdown) is available.

Reviewer: ```suggestion
        - **Astro-Flavored Markdown** of GCN Circulars provides an enhanced experience in using our GCN Circulars submission web form.  It now optionally utilizes a custom form of Markdown, a lightweight markup language.  It allows embedding of data via tables, enriched text formatting, enhancing the already implemented automatic cross-referencing to other Circulars and common astronomy archives. [Documentation on Astro Flavored Markdown](/docs/circulars/markdown) is available.
```

>         - **Notices Schema v4.0.0** for JSON-format GCN Notices is released.  Explore GCN Schema via the [Schema Browser](https://gcn.nasa.gov/docs/schema/v4.0.0/gcn/notices).  Changes to the GCN Schema include:

Reviewer: ```suggestion
        - **Notices Schema v4.0.0** for JSON-format GCN Notices is released.  Explore GCN Schema via the [Schema Browser](/docs/schema/v4.0.0/gcn/notices).  Changes to the GCN Schema include:
```

          - Addition of Einstein Probe WXT Notice topic
          - Addition of BurstCube Notice topic (not yet producing notices)

>           - Change to localization core schema that simplifies localization uncertainty parameterization

Reviewer: We had better explain that.

## 2024

    <CollectionItem
      className="maxw-none"
      variantComponent={
        <CollectionCalendarDate datetime={'May 20, 2024'} />
      }
    >
      <CollectionHeading headingLevel="h3">
        <Anchor>New Einstein Probe Notices, Astro Flavored Markdown for Circulars, and Notices Schema v4.0.0</Anchor>
      </CollectionHeading>
      <CollectionDescription>

> 

Reviewer: Add a short introductory sentence here... something to the effect of "Lots of new features this month!"

          - Occurrence Rate: The EP notices occur at a rate of ~100 times per year.
          - Time Delay: The post-event processing and communication of alert is distributed to GCN within ~60 seconds of detection.
          - Notice Type: The [new WXT notice type (`gcn.notices.einstein_probe.wxt.alert`)](/missions/einstein_probe) provides trigger information including localization.
          - Notice Schema and Examples: The JSON schema defining the notice format and examples can be viewed in the [Schema Browser](/docs/schema) or the GCN [schema repository](https://github.com/nasa-gcn/gcn-schema).

        #### Astro Flavored Markdown
         The GCN Circular web form now provides an enhanced experience a custom form of Markdown, a lightweight markup language.  It allows embedding of data via tables, enriched text formatting, enhancing the already implemented automatic cross-referencing to other Circulars and common astronomy archives. [Documentation on Astro Flavored Markdown](/docs/circulars/markdown) is available.

        #### Notices Schema v4.0.0

>         Explore GCN Schema via the [Schema Browser](/docs/schema/v4.0.0/gcn/notices).  Changes to the GCN Schema include:

Reviewer: ```suggestion
        Explore GCN Schema via the [Schema Browser](/docs/schema/stable/gcn/notices).  Changes to the GCN Schema include:
```
